editor/scene_create_dialog.cpp

```diff
@@ -131,6 +131,15 @@ void SceneCreateDialog::update_dialog() {
 	root_name = root_name_edit->get_text().strip_edges();
 	if (root_name.is_empty()) {
 		root_name = scene_name.get_file().get_basename();
+
+		if (root_name.is_empty()) {
+			// If the user clears the scene name make sure we reset the placeholder text.
```

Comment:
If you remove the other `set_placeholder()` line, this comment should be reworded or removed.